src/orders/groupOrder.js

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable import/no-cycle */
+import axios from "axios";
+import { retreiveCachedItems } from "../core/services.js";
+import { renderOffers, renderProducts } from "../products/renderProducts.js";
+import client from "../server.js";
+
+import { BASEURL } from "../core/urls.js";
+import { menus } from "../menus/menuoptions.js";
+import {
+  askForQuantity,
+  renderProductCategories,
+} from "../users/buyer/buyermenus.js";
+
+export const checkIfUserIsInGroup = async () => {
+  const user = await retreiveCachedItems(client, ["user_id"]);
+  const data = {
+    // TODO: Custom ID
+    user_id: user,
+  };
+  const response = await axios.post(`${BASEURL}/ussd/isuseringroup`, data);
+  if (response.data.status === "success") {
+    return response.data.group_ID;
+  }
+  return false;
+};
+
+export const actionToTake = async (state) => {
+  let message = "";
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/JoeMindo/USSDTest/pullRequest?prid=8695127)
```suggestion
  let message = '';
```